features.md

```diff
@@ -2,8 +2,10 @@
 
 1) Privide a GUI for configuring and running program
 
-2) Keep track of each color used, and output an image that idetifies each primitice with a number correcponding to the color of that primitive.
-  A) As an App user, I want to produce an image with numbers identifying each color so that I can produce "Paint by Numbers" and a person can maunually reproduce the image by filling each primitive with the numbered color.
-  B) 
+2) Keep track of each color used, and output an image that idetifies each primitive with a number correcponding to the color of that primitive.
+
+    A) As an App user, I want to produce an image with numbers identifying each color so that I can produce "Paint by Numbers" and a person can maunually reproduce the image by filling each primitive with the numbered color.
```

Comment:
Is the User Story as clear as it could be. Will a reader unfamiliar with the project understand it?